link.x

```diff
@@ -39,6 +41,18 @@ PROVIDE(ExceptionHandler = DefaultExceptionHandler);
 
 SECTIONS
 {
+    .flash_config _flash_config:
```

Comment:
I prefer not to include this in the linker script.
REGION_HEADER is meaningless for RAM, XPI0_APP only project.

Reply:
Do you mean `flash_config` only, or together with `boot_header`?